### PR TITLE
Add create-project example to the quickstart

### PR DESCRIPTION
**Pull request by helfi92**
Reference: https://github.com/mozilla-neutrino/neutrino-dev/issues/606

### CODE REVIEW

**Comment by helfi92** [comment]
This first commit adds documentation for the `react` preset. If I can get an OK, I will add similar docs to the other relevant presets.

**Comment by helfi92** [comment]
Added quickstart documentation to the other presets.

File: docs/packages/library/README.md [hunk 1]
```diff
@@ -61,6 +61,39 @@ project. This includes JavaScript files that would be available to your compiled
 
 ## Quickstart
 
+The fasted way to get started is by using the `create-project` scaffolding tool.
```
Comment (eliperelman) [inline docs/packages/library/README.md]: s/fasted/fastest/

File: docs/packages/library/README.md [hunk 2]
```diff
@@ -61,6 +61,39 @@ project. This includes JavaScript files that would be available to your compiled
...
+Don’t want to use the CLI helper? No worries, we got you covered with the [manual installation](#manual-installation).
```
Comment (eliperelman) [inline docs/packages/library/README.md]: Either "No worries; we have got you covered" or "No worries; we have you covered"

File: docs/packages/library/README.md [hunk 3]
```diff
@@ -61,6 +61,39 @@ project. This includes JavaScript files that would be available to your compiled
...
+```
+❯ yarn create @neutrinojs/project <directory-name>
+```
+
+_Note: The `create` command is a shorthand that helps you do two things at once. See the [Yarn create docs](https://yarnpkg.com/lang/en/docs/cli/create) for more details._
+
+#### npm/npx
+
+[`npx`](https://github.com/zkat/npx) comes pre-installed with `npm`. If you’re running an older version of `npm`, then
+`npm install -g npm` to update to the latest version. Long story short, `npx` allows you to keep your
```
Comment (eliperelman) [inline docs/packages/library/README.md]: Remove the sentence "Long story short...".

**Comment by helfi92** [comment]
Oops, I will need to fix the relative links in `packages/`. Will do that now.

**Comment by helfi92** [comment]
Done.

**Pull request merged**